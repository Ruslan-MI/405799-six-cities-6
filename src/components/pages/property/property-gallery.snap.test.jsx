import React from "react";
import {
  render,
} from "@testing-library/react";
import PropertyGallery from "./property-gallery";
import {
  mockOffer,
} from "../../../mocks-for-test";

it(`Property gallery should render correctly`, () => {

  const {
    container,
  } = render(<PropertyGallery images={mockOffer.images} />);

  expect(container).toMatchSnapshot();
});
