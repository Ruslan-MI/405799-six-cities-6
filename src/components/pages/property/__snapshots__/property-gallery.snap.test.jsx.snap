// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Property gallery should render correctly 1`] = `
<div>
  <div
    class="property__gallery-container container"
  >
    <div
      class="property__gallery"
    >
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="img/1.png"
        />
      </div>
      <div
        class="property__image-wrapper"
      >
        <img
          alt="Photo studio"
          class="property__image"
          src="img/2.png"
        />
      </div>
    </div>
  </div>
</div>
`;
